data/README.md

```diff
@@ -0,0 +1,43 @@
+# Data sources
```

Comment:
I got confused by this file and the whole folder. Is `download_data_for_preprocessing.py` downloading all missing files here? If yes, please mention it somewhere.

Reply:
Described in read me